cmd/algokey/multisig.go

```diff
@@ -109,8 +109,8 @@ var multisigCmd = &cobra.Command{
 	},
 }
 
-var convertCmd = &cobra.Command{
-	Use:   "convert -t [transaction file] -p \"[threshold] [PK1] [PK2] ...\"",
```

Comment:
I'd like to see the help be more explicit on the address ordering:

	Use:   "append-auth-addr -t [transaction file] -p \"[threshold] [Address 1] [Address 2] ... [Address n]\"",